plugins/coral-plugin-offtopic/client/components/OffTopicCheckbox.js

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
 import styles from './styles.css';
-
 import {t} from 'plugin-api/beta/client/services';
+import {PLUGIN_NAME, DEFAULT_CONFIG} from '../constants';
 
 export default class OffTopicCheckbox extends React.Component {
 
+  config = this.props.config;
+  pluginConfig = {...DEFAULT_CONFIG, ...(this.config && this.config[`${PLUGIN_NAME}`])};
```

Comment:
Reasoning: The config is coming from the props, which can change over time. Turning this into the function above will always calculate the config from the most current props.